src/main/java/org/mineacademy/fo/plugin/AutoRegisterScanner.java

```diff
@@ -60,7 +53,7 @@ final class AutoRegisterScanner {
 	/**
 	 * Automatically register the main command group if there is only one in the code
 	 */
-	private static List<SimpleCommandGroup> registeredCommandGroups = new ArrayList<>();
+	private static final List<SimpleCommandGroup> registeredCommandGroups = new ArrayList<>();
```

Comment:
Dakujem, ale ked vkladas Pull Request, dolezite je mat len tie zmeny, ktore suvisia s tvojou funkciou.

Tu vidim ze je poukladane "final" a tiez medzery a formatovatie v celej classe ine, ako to, co tam bolo, tym padom je horsie vidiet, ake su skutocne zmeny.

Reply:
V poriadku.. čo sa týka toho pull requestu.. nekomentujem to, save actions plugin, všimol som si to až neskôr.